wezterm-gui/src/overlay/copy.rs

```diff
@@ -1160,12 +1201,23 @@ pub fn copy_key_table() -> KeyTable {
         (
             WKeyCode::Char(' '),
             Modifiers::NONE,
-            KeyAssignment::CopyMode(CopyModeAssignment::ToggleSelectionByCell),
+            KeyAssignment::CopyMode(CopyModeAssignment::SetSelectionMode(Some(
+                SelectionMode::Cell,
+            ))),
         ),
         (
             WKeyCode::Char('v'),
             Modifiers::NONE,
-            KeyAssignment::CopyMode(CopyModeAssignment::ToggleSelectionByCell),
+            KeyAssignment::CopyMode(CopyModeAssignment::SetSelectionMode(Some(
+                SelectionMode::Cell,
+            ))),
+        ),
+        (
+            WKeyCode::Char('V'),
+            Modifiers::NONE, // FIXME: Seems to work with NONE, but why G also have SHIFT ?
```

Comment:
Some platforms have SHIFT set when the key is uppercase, but some do not :-/

Reply:
Weird.. well, I added the mapping for both